.github/workflows/linux.yml

```diff
@@ -24,56 +24,81 @@ on:
       - master
 
 concurrency:
-  group: ${{ github.head_ref || github.run_id }}-linux
+  group: ${{ github.ref || github.run_id }}-linux
   cancel-in-progress: true
 
 jobs:
   Build:
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-20.04-8-cores
+    runs-on: aks-linux-16-cores
```

Comment:
Do you switch only build job to self hosted runner? What is about FW Hub Tests?

Reply:
I want to switch this pipeline first to start to gather data on real-life load. If everything will be ok with these runners, we can start to switch other pipelines as well.